language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SortingUtil.java

```diff
@@ -250,4 +254,85 @@ public static String genSortText(int rank) {
 
         return Optional.empty();
     }
+
+    public static void toDefaultSorting(BallerinaCompletionContext context, List<LSCompletionItem> completionItems) {
+        for (LSCompletionItem item : completionItems) {
```

Comment:
@nadeeshaan does this address the following scenario. In a top-level, `function` keyword and `public function main` snippet sorted to list together?

Reply:
No, this is for the initial default sorting, and the mentioned use-case have to handle separately in each context's sorting logic.